src/formatting_stack/reporters/pretty_line_printer.clj

```diff
@@ -0,0 +1,78 @@
+(ns formatting-stack.reporters.pretty-line-printer
```

Comment:
Feel free to reference github.com/xojs/xo in the ns doc

...and similarly feel free to reference this ns from https://github.com/nedap/formatting-stack#consolidated-reporting or such (to keep these features discoverable)